README.md

```diff
@@ -1,2 +1,35 @@
 # helper-bot
+05/19/2021
+
+
 Use Discord Slash Commands to offer people helpful advice
+
+To add your bot to your discord server with the appropriate permissions, ensure that the permission for applications.commands exists in addition to the bot OAUTH scope. Without this you will receive 
+
+    "403 Forbidden (error code: 50001): Missing Access"
+
+
+Dependencies: 
+
+    pip install discord.py
+    pip install git+https://github.com/eunwoo1104/discord-py-slash-command.git 
+    *discord-py-slash-command requires the latest release from git as the bot uses features that aren't available on the Python Package Index release*
```

Comment:
Maybe make use of `pip install -r requirements.txt`

Reply:
yea, just wasn't sure if the git+https://github.com/eunwoo1104/discord-py-slash-command.git would still work in requirements.txt because just pulling the latest via pip install -r requirements.txt does not work with this code as it uses features that aren't released yet.